src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5680,14 +5618,6 @@ void CalcThermalResilience(EnergyPlusData &state)
 void ReportThermalResilience(EnergyPlusData &state)
 {
 
-    int HINoBins = 5;                     // Heat Index range - number of bins
-    int HumidexNoBins = 5;                // Humidex range - number of bins
-    int SETNoBins = 5;                    // SET report column numbers
-    int ColdHourOfSafetyNoBins = 5;       // Cold Stress Hour of Safety number of columns
-    int HeatHourOfSafetyNoBins = 5;       // Heat Stress Hour of Safety number of columns
-    int UnmetDegreeHourNoBins = 6;        // Unmet Degree Hour number of columns
-    int DiscomfortWtExceedHourNoBins = 4; // Unmet Degree Hour number of columns
```

Comment:
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5683]:(style),[variableScope],The scope of the variable 'HINoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5684]:(style),[variableScope],The scope of the variable 'HumidexNoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5685]:(style),[variableScope],The scope of the variable 'SETNoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5686]:(style),[variableScope],The scope of the variable 'ColdHourOfSafetyNoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5687]:(style),[variableScope],The scope of the variable 'HeatHourOfSafetyNoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5688]:(style),[variableScope],The scope of the variable 'UnmetDegreeHourNoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:5689]:(style),[variableScope],The scope of the variable 'DiscomfortWtExceedHourNoBins' can be reduced.